### PR TITLE
Box Viro Wants Plasma v2

### PR DESCRIPTION
**Pull request by ChrisDirksDev**
:cl:
tweak: 2 plasma sheets have been added to viro break room on box
/:cl:

[why]: Viro's on box want plasma in viro so they dont have to go raid it from other departments

### CODE REVIEW

**Comment by SgtxRootbeer** [comment]
there's a bottle of plasma in the smartfridge, use it, hmmm?

**Comment by Crexfu** [comment]
oh is there, i assumed like all the other jobs you needed to grind it yourself, if thats the case then this isnt needed. Although now i know how to do map pr's so wew I guess

**Comment by XDTM** [comment]
Having viro in line with all other maps wouldn't be awful, though.

**Comment by MoreRobustThanYou** [comment]
Extra plasma won't hurt, especially if you accidentally mix your first batch with virus food

**Comment by Steelpoint** [comment]
This is fine, all other viro's have this on offer.

The real gatekeeper of viro is Uranium.

**Review by Cyberboss — APPROVED** [review]

File: _maps/map_files/BoxStation/BoxStation.dmm [hunk 1]
```diff
@@ -41535,6 +41535,12 @@
 /obj/machinery/newscaster{
 	pixel_x = -30
 	},
+/obj/item/stack/sheet/mineral/plasma{
+	layer = 2.9
+	},
+/obj/item/stack/sheet/mineral/plasma{
+	layer = 2.9
```
Comment (Cyberboss) [inline _maps/map_files/BoxStation/BoxStation.dmm]: @ShizCalev are these layer edits necessary?

Comment (ShizCalev) [inline _maps/map_files/BoxStation/BoxStation.dmm]: No, but it's present on all the others on the map too apparently. I'll add it to my list of things to clean up.

**Pull request merged**